utils/test-connector-remote/src/main/java/de/sovity/edc/extension/e2e/connector/DataTransferTestUtil.java

```diff
@@ -43,6 +44,15 @@ public static JsonObject buildDataAddressJsonLd(String baseUrl, String method) {
                 .build();
     }
 
+    public static Map<String, String> buildDataAddressProperties(String baseUrl, String method) {
+        return Map.of(
+                EDC_NAMESPACE + "type", "HttpData",
+                EDC_NAMESPACE + "baseUrl", baseUrl,
+                EDC_NAMESPACE + "method", method
```

Comment:
- [ ] create issue: migration of documentation
- [ ] create issue: migration of UI (data address properties)